### DISCUSSION TITLE
Brand New Watermeter won´t read Pointers

### DISCUSSION THREAD

**Discussion by mikepiko** [jomjol/AI-on-the-edge-device#1137] [Q&A]
Just got a Brand New "zeroed" Watermeter. 
The old one got recognized completely but the new one i have Problems that the Pointers are not Recognized right:

![image](https://user-images.githubusercontent.com/38277702/194147874-83233530-9856-41e5-bdf9-d90f90fbf56a.png)

The Digital First Two i can understand because they are not Clearly visible currently.

The Problem are the red Analog Pointers ... the Right one "0.1" is on Digit 2 but it got recognized as a 7.
The Second one is between 7 and 8 and got read as 1 ... 

Any Idea how i can Fix that ?

**Comment by ghost** [comment]
Against the reflection rotate the housing a bit anit clockwise, so that the reflection is at the black wheel.

Read https://github.com/jomjol/AI-on-the-edge-device/wiki/ROI-Configuration for the digits.

The sharpness could be better, but is not the reason.

Check if the analog pointer are in order. The name is not decisive here. 

Which model you use? Could a give a screenshot of the recognition page?
